ui/src/app/app.component.html

```diff
@@ -0,0 +1,74 @@
+<nav class="navbar navbar-expand-md navbar-dark bg-dark">
+  <a class="nav-link" href="#">Payment Simulator</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse"
+          aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarCollapse">
+    <ul class="navbar-nav mr-auto">
+      <!--<li class="nav-item">
+        <a class="nav-link" routerLink="/example" routerLinkActive="active">Example</a>
+      </li>-->
+    </ul>
+  </div>
+</nav>
+<div class="container">
+  <br>
+  <br>
+  <br>
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="form-group landingLabels">
+          <label><b>Merchant</b></label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-group">
+          <label>PAYGATE - TEST SYSTEM</label>
+        </div>
+      </div>
+      <div class="w-100"></div>
+      <div class="col">
+        <div class="form-group landingLabels">
+          <label><b>Reference</b></label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-group">
+          <label>Paygate Test</label>
+        </div>
+      </div>
+      <div class="w-100"></div>
```

Comment:
Why a blank column? Wouldn't just starting a new `.row` solve this?

Reply:
Again, design session. This is something I took off bootstrap components to wrap col and it looked nice.